repository-data/development/src/main/resources/hcm-content/content/gallery/publicationsystem.yaml

```diff
@@ -1,5 +1,6 @@
 ---
 /content/gallery/publicationsystem:
+  .meta:order-before: ipad-landscape-v2.png
```

Comment:
Will this cause a problem because the `ipad-landscape-v2.png` is in development but this file is in application?

Reply:
I think it has been added while auto-exporting. I think it can be removed.